client/verta/verta/client.py

```diff
@@ -932,6 +932,23 @@ def get_or_create_endpoint(self, path=None, description=None, workspace=None, id
 
 
     def get_endpoint(self, path=None, workspace=None, id=None):
+        """
+        Retrieves an already created Endpoint. Only one of path or id can be provided.
```

Comment:
I like to use backticks in the docstring's summary whenever referring to parameters.
```suggestion
        Retrieves an already created Endpoint. Only one of `path` or `id` can be provided.
```

Reply:
Fixed.